parsing/src/main/scala/org/apache/pekko/macros/LogHelper.scala

```diff
@@ -13,11 +13,8 @@
 
 package org.apache.pekko.macros
 
-import org.apache.pekko
-import pekko.annotation.InternalApi
-import pekko.event.LoggingAdapter
-
-import scala.reflect.macros.blackbox
+import org.apache.pekko.annotation.InternalApi
```

Comment:
maybe fix this import - move with rest of pekko ones and remove org.apache bit

Reply:
Fixed and pushed